packages/webdriverio/tests/chainablePromiseArray.test.ts

```diff
@@ -0,0 +1,30 @@
+import { describe, it, expect } from 'vitest'
+import type { ChainablePromiseArray } from '../src/types.js'
+
+describe('ChainablePromiseArray', () => {
+    it('entries should return an async iterator of key/value pairs', async () => {
+        const elements = [
+            { elementId: 'elem1' },
+            { elementId: 'elem2' },
+            { elementId: 'elem3' }
+        ]
+        const chainablePromiseArray: ChainablePromiseArray = {
+            entries: async function* () {
+                for (const [index, element] of elements.entries()) {
+                    yield [index, element]
+                }
+            }
+        } as any
```

Comment:
This test is not really useful as you are testing only if `entries` is a function of the `ChainablePromiseArray` type. A unit test for this is not really needed. Let's enhance our type tests in `/tests/typings/webdriverio/async.ts` and verify the function can be called.

Reply:
Should be done